xds/src/main/java/io/grpc/xds/XdsResourceType.java

```diff
@@ -38,7 +39,7 @@
 import java.util.Set;
 import javax.annotation.Nullable;
 
-abstract class XdsResourceType<T extends ResourceUpdate> {
+public abstract class XdsResourceType<T extends ResourceUpdate> {
```

Comment:
mark as @experimentalAPI("*issue link*")
Then create an issue tracking "XdsResourceType is experimental"

Reply:
Done.